junit-jupiter-params/src/main/java/org/junit/jupiter/params/ParameterizedClassContext.java

```diff
@@ -155,9 +153,6 @@ private static <A extends Annotation> List<ArgumentSetLifecycleMethod> findLifec
 		List<Method> methods = findAnnotatedMethods(testClass, annotationType, traversalMode);
 
 		return methods.stream() //
-				.filter(ModifierSupport::isNotPrivate) //
-				.filter(testInstanceLifecycle == PER_METHOD ? ModifierSupport::isStatic : __ -> true) //
-				.filter(ReflectionUtils::returnsPrimitiveVoid) //
```

Comment:
:speech_balloon: This is already validated in `junit-jupiter-engine` so there's no need to do so again.